ledger-api/grpc-definitions/com/daml/ledger/api/v2/package_service.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v1/package_service.proto";
+
+option java_outer_classname = "PackageServiceOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
+// Allows clients to query the Daml-LF packages that are supported by the server.
+service PackageService {
+
+  // Returns the identifiers of all supported packages.
+  rpc ListPackages (ListPackagesRequest) returns (com.daml.ledger.api.v1.ListPackagesResponse);
```

Comment:
That is a good idea, 
* Although somewhat contradicts what we decided with all the rest of the streaming responses (singular). 
* Also in case of packages it is just a package-id. We could probably go a long way with one message still.
* For parties we have an admin endpoint on the ledger-api. Didn't we said we probably will use the canton admin endpoints in the future?
Let's discuss this.

Reply:
As discussed: we postpone improvements until 3.0